sensor_msgs/PointCloud2 cloud_in
---
sensor_msgs/PointCloud2[] cluster_cloud_list
